galaxy_importer/ansible_test/container/eda/tox.ini

```diff
@@ -1,19 +1,12 @@
 [tox]
 requires = 
     ruff
-    darglint
     pylint
 
 
 [testenv:ruff]
 deps = ruff
-commands = - ruff check --select ALL --ignore INP001,FA102,UP001,UP010,I001,FA100,PLR0913,E501 -q {posargs}/extensions/eda/plugins
-
-
-[testenv:darglint]
-deps = darglint
-commands = - darglint -s numpy -z full {posargs}/extensions/eda/plugins
-
+commands = - ruff check --select ALL --ignore D100,D104,INP001,FA102,UP001,UP010,I001,FA100,PLR0913,E501 -q {posargs}/extensions/eda/plugins
```

Comment:
Have you try to declare these exceptions in more reusable way, like in pyproject? If it works, we could ensure they are applied at project level, outside tox.

Reply:
forget it. I did not see the project, I was thinking it was our collection xD